README.md

```diff
@@ -40,3 +40,44 @@ Options:
       --[no-]warning    Shows the warning message before potentially destructive operation.
                         (defaults to on)
 ```
+
+## Scoring
+
+### Health score
+
+A package gets `0.0` if any major analyzer process fails (e.g. `pub upgrade`,
+`dartanalyzer` or `dartfmt`).
+
+Otherwise the score starts with `1.0`, and
+- analyzer errors reduce it by 25%
+- analyzer warnings reduce it by 5%
+- analyzer hints reduce it by 0.5%
+- platform conflicts reduce it by 0.25 points (absolute reduction)
+
+`health = 0.75^errors * 0.95^warnings * 0.995^hints - 0.25*conflicts`
```

Comment:
Possibly mention it can't get below 0.0.

Reply:
I think that is from the next line, and unlikely to cause confusion (I hope :))